governance-with-multisig/truffle-config.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const ROPSTEN_HOST = null;
+const ROPSTEN_PORT = null;
+
+module.exports = {
+  networks: {
+    local: {
+      host: 'localhost',
+      port: 9545,
+      gas: 5000000,
+      network_id: '*'
+    },
+    ropsten: {
+      host: ROPSTEN_HOST,
+      port: ROPSTEN_PORT,
+      gas: 4000000,
+      network_id: 3,
+      gasPrice: 10e9
+    }
```

Comment:
do we need `ropsten`?

Reply:
The supporting blog post works on `ropsten`